tox.ini

```diff
@@ -14,8 +14,7 @@ commands = flake8 barril
 
 [testenv]
 passenv = TOXENV CI TRAVIS TRAVIS_* APPVEYOR APPVEYOR_*
-deps =
-    -r{toxinidir}/requirements_dev.txt
+deps = .[testing]
```

Comment:
I think we should be using [extras](https://tox.readthedocs.io/en/latest/config.html?highlight=extras#confval-extras=MULTI-LINE-LIST) instead; what you did here works, but I think you are reinstalling `barril` using `pip`, instead of installing `barril` from *the package* created by `tox`. 👍

Reply:
Went ahead and applied the change myself. 😁